sycl/test/basic_tests/half_type.cpp

```diff
@@ -193,8 +193,8 @@ inline bool bitwise_comparison_fp32(const half val, const uint32_t exp) {
 }
 
 int main() {
-  // We assert that the length is 1 because we use env to select the device
-  assert(device::get_devices().size() == 1);
+  // We assert that the length is not 0 because we run it for existing device types
+  assert(device::get_devices().size() > 1);
```

Comment:
I suggest just using a default device selector instead of device::get_devices() in this test, there'll be no need for this assertion then.